libmscore/rendermidi.cpp

```diff
@@ -748,7 +748,7 @@ void MidiRenderer::collectMeasureEventsDefault(EventMap* events, Measure* m, con
 //    redirects to the correct function based on the passed method
 //---------------------------------------------------------
 
-void MidiRenderer::collectMeasureEvents(EventMap* events, Measure* m, const StaffContext& sctx, int tickOffset)
+void MidiRenderer::collectMeasureEvents(EventMap* events, Measure const * m, const StaffContext& sctx, int tickOffset)
```

Comment:
Correct me if I'm wrong, but don't we prefer `const Measure*` over `Measure const *`? Or maybe that's just an inconsequential thing, I don't know :)

Reply:
I believe the order wasn't specified in the "old" coding rules in effect on 3.x

The new ones indeed write `const T*` as a preference; but I'm still fighting that rule from a consistency pov, One should read type declarations from right to left and that would mean language natural that `T const *` makes more sense.
Anyhow.. let's not get into that discussion here.

If it must be changed to abide by coding rules, then I'll make that change; if I don't *have* to, then I won't ;-)

Reply:
Yeah, please don't feel you have to just because I've piped up ;)

Reply:
wow, I just realised that jeetee and jthistle were not the same person after all 😆 
All that time I thought both were a short for johan temmerman, pointing to the same person...

Reply:
@Marr11317 Who says we're not the same person? 😇 
I'm that talkative one that posts in the forums and occasionally fixes something.
James is the one that quietly fixes a bunch of stuff and occasionally speaks up.

Reply:
And I believe there's a certain difference in age. 😉

Reply:
Not a big one